tdrs-backend/Pipfile

```diff
@@ -4,8 +4,8 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-awscli = "*"
-awscli-local = "*"
+awscli = "==1.25.82"
+awscli-local = "==0.20"
```

Comment:
I pinned the `awscli` and `awscli-local` versions to avoid potential conflicts. these libs generally ship with `boto3` updates, which we had pinned the version of.